### DISCUSSION TITLE
Images not appearing in Storybook

### DISCUSSION THREAD

**Discussion by BenPughyy** [jsakamoto/BlazingStory#57] [Q&A]
### Hi!

I'm using your BlazingStory framework on my project, but when I'm using components that have images in them they aren't appearing. *I can't find any documentation on how to fix this*, could you please help? The images are contained in **Project.One's wwwroot** folder, and then the Storybook is it's own Project within the same Solution.

If they is an import or something I am missing, please let me know.

Thanks.

**Comment by jsakamoto** [comment]
Hi @BenPughyy,

Unfortunately, I've not succeeded in reproducing the behavior you reported. I attached a Blazor WebAssembly app and its Stories project to this thread, which I am using to try to reproduce the problem on my side.

[📦BlazorWasmApp1.zip](https://github.com/user-attachments/files/17156981/BlazorWasmApp1.zip)

I implemented the Blazing Story to be able to handle these kinds of scenarios, so it must work well. But as you said, if it didn't, I might have missed something.

Could you provide a project that can reproduce the problem?

**Reply by BenPughyy** [reply]
Hi @jsakamoto ,

Apologies, ignore me.
I had the wrong image path referenced, nothing to do with an error on Blazing Stories :) sorry to waste your time.

Thanks.

**Reply by jsakamoto** [reply]
Hi @BenPughyy,
No problem! I was relieved to hear that.
Have a happy coding :)
